controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -187,6 +187,42 @@ bool master_thread::socket_disconnected(Socket *sd)
 
 bool master_thread::handle_cmdu(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
 {
+    // Note: sd can be equal to nullptr it the master_thread is derived from btl_local_bus and the
```

Comment:
The part `it the master_thread is derived from btl_local_bus` is not clear and IMO redundant.  Actually, the entire comment is redundant - we're only using `sd` if `from_bus` returns false, so all is well...

Reply:
Removed the comment.